stellar/txobserver/main.go

```diff
@@ -36,8 +41,14 @@ func NewObserver(client *horizonclient.Client, store *store.DB) *Observer {
 	return o
 }
 
+func (o *Observer) GetLastLedgerCloseTime() (time.Time, error) {
+	// TODO error if not synced?
```

Comment:
OK, I need to rebuild this part anyway because I just realized that in case of frontend/backend deployment the frontend instances won't have access to `Observer`. And `GetLastLedgerCloseTime` method is there specifically to support HTTP requests.

Reply:
OK, fixed in 5b4a9589bb084f079c1843c5112e1a5bff130568.